src/view/actions/resetEventMergeId.jsx

```diff
@@ -40,60 +37,28 @@ const validationSchema = object().shape({
     .matches(singleDataElementRegex, DATA_ELEMENT_REQUIRED)
 });
 
-const ResetEventMergeId = ({
-  initInfo,
-  formikProps,
-  registerImperativeFormApi
-}) => {
-  useEffect(() => {
-    registerImperativeFormApi({
-      getSettings,
-      formikStateValidationSchema: validationSchema
-    });
-    formikProps.resetForm({ values: getInitialValues({ initInfo }) });
-  }, []);
-
-  // Formik state won't have values on the first render.
-  if (!formikProps.values) {
-    return null;
-  }
-
-  return (
-    <FormElementContainer>
-      <DataElementSelector>
-        <FormikTextField
-          data-test-id="eventMergeIdField"
-          name="eventMergeId"
-          label="Event Merge ID"
-          description="Please specify the data element that represents the event merge ID you would like to reset."
-          width="size-5000"
-          isRequired
-        />
-      </DataElementSelector>
-    </FormElementContainer>
-  );
-};
-
-ResetEventMergeId.propTypes = {
-  initInfo: PropTypes.object,
-  formikProps: PropTypes.object,
-  registerImperativeFormApi: PropTypes.func
-};
-
-const ResetEventMergeIdView = () => {
+const ResetEventMergeId = () => {
   return (
     <ExtensionView
-      render={() => {
-        return (
-          <ExtensionViewForm
-            render={props => {
-              return <ResetEventMergeId {...props} />;
-            }}
+      getInitialValues={getInitialValues}
+      getSettings={getSettings}
+      formikStateValidationSchema={validationSchema}
+      render={() => (
+        <FormElementContainer>
+        <DataElementSelector>
```

Comment:
Looks like indentation needs adjusted. If you run `npm run format`, you'll see one other adjustment in a different file as well. Running `npm run lint` will remove a couple unused imports as well.